### PR TITLE
feat: add item and assets permissions validation

### PR DESCRIPTION
**Pull request by gabrielfs7**
feat: add item and assets permissions validation

https://oat-sa.atlassian.net/browse/ADF-573

### CODE REVIEW

File: actions/class.ItemContent.php [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 use oat\generis\model\OntologyAwareTrait;
 use oat\tao\helpers\FileUploadException;
+use oat\tao\model\accessControl\Context;
```
Comment (boajer) [inline actions/class.ItemContent.php:24]: AFAIK this has been added at core `v48.22.1`

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: actions/class.ItemContent.php (hunk 1 above)
Comment (gabrielfs7) [reply actions/class.ItemContent.php:24]: I will update with new tao-core version after this PR: https://github.com/oat-sa/tao-core/pull/3048

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**